File: hr-management-fe/utils/validate.ts
export function isValidGoogleCoordinate(s: string): boolean {
    // Regular expression for Google coordinates (latitude, longitude)
    const regex = /^[-+]?\d+(?:\.\d+)?,\s*[-+]?\d+(?:\.\d+)?$/

    // Match the string against the regular expression
    return regex.test(s)
}

File: backend/pkg/validate/validate.go
package validate

import "regexp"

func IsValidGoogleCoordinate(s string) bool {
	// Regular expression for Google coordinates (latitude, longitude)
	regex := `^[-+]?\d+(?:\.\d+)?,\s*[-+]?\d+(?:\.\d+)?$`

	// Match the string against the regular expression
	return regexp.MustCompile(regex).MatchString(s)
}
